### PR TITLE
Feature/neftymarket stats

### PR DESCRIPTION
**Pull request by jandrad**

### CODE REVIEW

**Review by javier3448 — APPROVED** [review]

File: src/api/namespaces/neftymarket/handlers/assets.ts [hunk 1]
```diff
@@ -3,10 +3,23 @@ import { NeftyMarketContext } from '../index';
 import { fillAssets } from '../../atomicassets/filler';
 import { buildAssetFillerHook, formatListingAsset } from '../format';
 import { getRawAssetsAction } from '../../atomicassets/handlers/assets';
+import QueryBuilder from '../../../builder';
+import {filterQueryArgs} from '../../validation';
 
 export async function getMarketAssetsAction(params: RequestValues, ctx: NeftyMarketContext): Promise<any> {
+    const args = filterQueryArgs(params, {
+        symbol: {type: 'string', default: null},
```
Comment (blood0cean) [inline src/api/namespaces/neftymarket/handlers/assets.ts:11]: why not default it to a configuration constant or something? doesn't work?

Comment (jandrad) [reply src/api/namespaces/neftymarket/handlers/assets.ts:11]: Because it depends on the chain

Comment (blood0cean) [reply src/api/namespaces/neftymarket/handlers/assets.ts:11]: yeah I meant like an environment variable

**Review by blood0cean — APPROVED** [review]
lgtm, tested the endpoints in tupaca02 and they work (without applying filters though)

File: src/api/namespaces/neftymarket/handlers/assets.ts (hunk 1 above)
Comment (jandrad) [reply src/api/namespaces/neftymarket/handlers/assets.ts:11]: It can be a configuration.

**Pull request merged**